petastorm/tests/test_spark_dataset_converter.py

```diff
@@ -243,3 +244,26 @@ def map_fn(_):
 
         result = self.spark.sparkContext.parallelize(range(1), 1).map(map_fn).collect()[0]
         self.assertEqual(result, 100)
+
+
+class TfConverterTestOnDelta(TfConverterTest):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
In `TfConverterTest`, add
```
    def tearDownClass(cls):
        cls.spark.stop()
```
and remove `def tearDown` method in `TfConverterTest`.